notion/collection.py

```diff
@@ -186,14 +186,40 @@ def get_schema_property(self, identifier):
                 return prop
         return None
 
-    def add_row(self, update_views=True, **kwargs):
+    def add_row(self, update_views=True, sourceBlockId=None, **kwargs):
```

Comment:
Would it be possible to use underscore-casing here, just to match the convention?